src/Microsoft.DotNet.Tools.Compiler.Csc/Program.cs

```diff
@@ -124,9 +125,10 @@ private static IEnumerable<string> GetDefaultOptions()
                 ? "-debug:full"
                 : "-debug:portable");
 
-            args.Add("-nowarn:CS1701");
-            args.Add("-nowarn:CS1702");
-            args.Add("-nowarn:CS1705");
+            foreach (var suppress in DefaultCompilerWarningSuppresses.Map["csc"])
+            {
+                args.Add($"-nowarn:{suppress}");
+            }
```

Comment:
You should change it. We hardcoded them before because we didn't have a way to represent them. Make that last change and this will be good.